edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/EdgeToDeviceMethodTest.cs

```diff
@@ -50,7 +50,7 @@ Task<MethodResponse> MethodHandler(MethodRequest methodRequest, object context)
                 await receiver.OpenAsync();
                 await receiver.SetMethodHandlerAsync("poke", MethodHandler, null);
 
-                await Task.Delay(TimeSpan.FromSeconds(5));
+                await Task.Delay(TimeSpan.FromSeconds(10));
```

Comment:
>n.FromSeconds(10)); [](start = 40, length = 19)

Missing the description on why of this change.

Reply:
added.